BlobStorage/Makefile

```diff
@@ -1,13 +1,15 @@
 export AZURE_REGION ?= westeurope
 export AZURE_RESOURCE_GROUP ?= AzureFunctionsCoralogix
-export AZURE_FUNCTION_NAME ?= coralogixblob
+export AZURE_FUNCTION_NAME ?= coralogixblob2
 export AZURE_STORAGE_CONNECTION ?= ""
 
 export CORALOGIX_PRIVATE_KEY ?= ""
 export CORALOGIX_APP_NAME ?= Azure
 export CORALOGIX_SUB_SYSTEM ?= blob
 export NEWLINE_PATTERN ?= (?:\r\n|\r|\n)
 
+export CORALOGIX_URL ?= "https://api.coralogix.com:443/api/v1/logs" # Change to .us / .in for other clusters
```

Comment:
The url is different between clusters I would suggest make it for now statically configured. And based on the environment point to the correct one in order to avoid mistakes

Reply:
what do you mean by statistically different?
originally we didn't provide any clear path to change, now we do ..